pkg/i2gw/providers/gce/extensions/input_extensions.go

```diff
@@ -1,92 +0,0 @@
-/*
```

Comment:
It looks like this PR is removing >2k lines of code specific to the GCE implementation, and removing the corresponding functionality. I think the minimum requirement for a PR that's doing as much refactoring as this one is to maintain the previous capabilities of ingress2gateway unless there's a clear reason we can't do that.